starknet_py/tests/e2e/docs/guide/test_sign_for_fee_estimate.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+
+@pytest.mark.asyncio
+async def test_signing_fee_estimate(gateway_account, map_contract):
+    account = gateway_account
+    # docs: start
+    # Create a transaction
+    call = map_contract.functions["put"].prepare(key=10, value=20, max_fee=1000)
```

Comment:
Is it necessary to show `max_fee` usage here?

Reply:
It isn't, removed